ratis-docs/src/site/markdown/configurations.md

```diff
@@ -199,6 +199,16 @@ treat the peer as caught-up. Increase this number when write throughput is high.
 | **Type**        | double, ranging from (0.0,1.0)                |
 | **Default**     | 0.9                                           |
 
+### Read After Write - Configurations related to read-after-write-consistency
+
+
+| **Property**    | `raft.server.read.read-after-write-consistent.write-index-cache.expiry-time`   |
```

Comment:
>     ******* interface org.apache.ratis.server.RaftServerConfigKeys$Read$ReadAfterWriteConsistent *******
>     constant: PREFIX = raft.server.read-after-write-consistent
>     key: raft.server.read-after-write-consistentwrite-index-cache.expiry-time (org.apache.ratis.util.TimeDuration, default=60s)

Above is the output of `RaftServerConfigKeys.main()`.  We should fix the conf key as below.

```java
+++ b/ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java
@@ -215,9 +215,9 @@ public interface RaftServerConfigKeys {
     }
 
     interface ReadAfterWriteConsistent {
-      String PREFIX = RaftServerConfigKeys.PREFIX + ".read-after-write-consistent";
+      String PREFIX = Read.PREFIX + ".read-after-write-consistent";
 
-      String WRITE_INDEX_CACHE_EXPIRY_TIME_KEY = PREFIX + "write-index-cache.expiry-time";
+      String WRITE_INDEX_CACHE_EXPIRY_TIME_KEY = PREFIX + ".write-index-cache.expiry-time";
       /** Must be larger than {@link Read#TIMEOUT_DEFAULT}. */
       TimeDuration WRITE_INDEX_CACHE_EXPIRY_TIME_DEFAULT = TimeDuration.valueOf(60, TimeUnit.SECONDS);
```

Reply:
Thanks for pointing out! Let me fix these in this patch.